jbpm-designer-backend/src/main/java/org/jbpm/designer/web/preprocessing/impl/JbpmPreprocessingUnit.java

```diff
@@ -618,9 +644,15 @@ public void evaluateWorkDefinitions(Map<String, WorkDefinitionImpl> workDefiniti
 
                 Map<String, Object> parameterValues = new HashMap<>();
                 if (workDefinitionMap.get("parameterValues") != null) {
+                    ClassLoader parentLoader = Thread.currentThread().getContextClassLoader();
```

Comment:
The error was occurring because the EnumDataType could not find the enumeration/class that was used in the constructor of the EnumDataType. Even when the artifact was added as a dependency for the project, the Workbench's class loader did not include it; consequently it could not find the enumeration/class. The mavenDependencies would seem to be a logical place for adding a reference to the required artifact. Expanding on the parent class loader, and passing it to the EnumDataType.getValueNames() method, allows the enumeration/class to be found.